src/ParameterDistributions.jl

```diff
@@ -771,71 +759,51 @@ Returns a concatenated mean of the parameter distributions.
 """
 mean(d::Parameterized) = mean(d.distribution)
 mean(d::Samples) = mean(d.distribution_samples, dims = 2)
-function mean(d::VectorOfParameterized)
-    return cat([mean(dd) for dd in d.distribution]..., dims = 1)
-end
-function mean(pd::ParameterDistribution)
-    return cat([mean(d) for d in pd.distribution]..., dims = 1)
-end
+mean(d::VectorOfParameterized) = reduce(vcat, mean.(d.distribution))
+mean(pd::ParameterDistribution) = reduce(vcat, mean.(pd.distribution))
```

Comment:
broadcast `mean` instead of looping. Then `reduce(vcat, ...)` instead of `cat(..., dims=1)`.